src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 57 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 0 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt && testexe -returncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e906
```powershell
At line:1 char:73
+ ... rncode 0 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt && testexe ...
+                                                                ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt && testexe -returncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e906
```powershell
At line:1 char:73
+ ... rncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt && testexe ...
+                                                                ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt || testexe -returncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e906
```powershell
At line:1 char:73
+ ... rncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement 'testexe -returncode 0 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt || testexe -returncode 1 > /tmp/d97a5290-07ca-4fb3-92c5-b132e906
```powershell
At line:1 char:73
+ ... rncode 0 > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/1.txt || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```
Experimental Feature: && and || operators.File redirection with && and ||.Handles redirection correctly with statement '(testexe -returncode 0 && testexe -returncode 1) > /tmp/d97a5290-07ca-4fb3-92c5-b132e9065a57/3.txt'
```powershell
At line:1 char:24
+ (testexe -returncode 0 && testexe -returncode 1) > /tmp/d97a5290-07ca ...
+                        ~~
The token '&&' is not a valid statement separator in this version.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Operators/BashControlOperator.Tests.ps1: line 207
207:             Invoke-Expression -Command $Statement

```